django_fsm/tests/test_abstract_inheritance.py

```diff
@@ -15,6 +15,17 @@ def publish(self):
         pass
 
 
+class AnotherFromAbstractModel(BaseAbstractModel):
+    """
+    This class exists to trigger a regression when multiple concrete classes
+    inherit from a shared abstract class (example: BaseAbstractModel).
+    Don't try to remove it.
```

Comment:
To be sure it does not get removed, you may add a small test in `TestinheritedModel` that makes sure that there are more than one subclass of the abstract class used for `InheritedFromAbstractModel`